Library/Formula/libpcap.rb

```diff
@@ -29,4 +22,25 @@ def install
   test do
     assert_match /lpcap/, shell_output("#{bin}/pcap-config --libs")
   end
+
+  # Availability.h was introduced in 10.5, guard it off
+  # https://github.com/the-tcpdump-group/libpcap/pull/1203
+  patch :p0, :DATA
```

Comment:
In the future, do you mind putting `patch` blocks further up in the file? They usually go under the source info, above dependencies. No big deal though.

Reply:
Yep, no problem.